### PR TITLE
[sdk/trace] support trace exporter that uses a batch processor and otlp exporter

### PR DESCRIPTION
**Pull request by moh-osman3**
**Description:** 
Adding support for collector based exporting (using batch processor and otlp exporter) for otel-go traces

**Testing:**
Successfully sent traces using a trace exporter from this change

### CODE REVIEW

File: lightstep/sdk/trace/exporters/otlp/otelcol/otlp.go [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+package otelcol
+
+import (
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/sdk/trace"
+)
+
+func copyAttributes(dest pcommon.Map, src attribute.Set) {
```
Comment (jmacd) [inline lightstep/sdk/trace/exporters/otlp/otelcol/otlp.go:24]: Will you add a TODO saying to consider creating an `internal` package somewhere near the top where the metrics and trace SDK can share this sort of common code?

**Review by jmacd — APPROVED** [review]
LGTM

**Review by jmacd — APPROVED** [review]
LGTM

File: lightstep/sdk/trace/exporters/otlp/otelcol/otlp.go (hunk 1 above)
Comment (moh-osman3) [reply lightstep/sdk/trace/exporters/otlp/otelcol/otlp.go:26]: Thanks for the review! Added the TODO

**Pull request merged**